src/coreclr/jit/codegenarmarch.cpp

```diff
@@ -669,7 +669,7 @@ void CodeGen::genIntrinsic(GenTree* treeNode)
     assert(varTypeIsFloating(srcNode));
     assert(srcNode->TypeGet() == treeNode->TypeGet());
 
-    // Right now only Abs/Ceiling/Floor/Truncate/Round/Sqrt are treated as math intrinsics.
+    // Right now only Abs/Ceiling/Floor/Truncate/Round/Sqrt/Max/Min are treated as math intrinsics.
```

Comment:
This comment is going to just keep growing. Might be better to remove it or change it to
```suggestion
    // Only a subset of functions are treated as math intrinsics.
```